stjude/proteinpaint
# Generate violin server route, api tests, and documentation

## Description
Use "augen" (automatic generator) with the scanned routes/*.ts files to:
- set up the server routes
- create unit tests
- generate API documentation

To test:
1. `npm run dev`
2. `./docs/build.sh`: should see a `termd/violin` section in http://localhost:3000/server.html
3. `npm run test:unit --workspace=server`: should see output for `server/termdb.violin.ts specs`, even though it may fail while examples are still missing

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
